### ISSUE TITLE
Raise Fluent::ConfigError along this repository's readme

### ISSUE THREAD

**Issue by tikkss** [fluent-plugins-nursery/fluent-plugin-windows-exporter#18]
Steps to reproduce
---

1. Install fluent-plugin-windows-exporter gem:
    ```sh
    % gem install fluent-plugin-windows-exporter
    ```
2. Edit Fluentd's configuration file along [Example Configuration](https://github.com/fluent-plugins-nursery/fluent-plugin-windows-exporter/blob/ba89041cdcd4ec51a76a6e4b5ef217ebde31c66e/README.md#example-configuration):
    ```sh
    % type test.conf
    <source>
      @type windows_exporter
      tag windows.metrics  # optional
      scrape_interval 60   # optional
      cpu true             # optional
      memory true          # optional
      os trues             # optional
      net true             # optional
      logical_disk true    # optional
    </source>
    <match windows.metrics>
      @type stdout
    </match>
    ```
3. Launch Fluentd:
    ```sh
    % fluentd -c test.conf
    ```

Expected result
---

Successfully launched Fluentd

Actual result
---

Error launching Fluentd with the below command.
(I omitted the leading date and time)

```
% fluentd -c test.conf
[info]: init supervisor logger path=nil rotate_age=nil rotate_size=nil
[info]: parsing config file is succeeded path="test.conf"
[info]: gem 'fluentd' version '1.15.3'
[info]: gem 'fluent-mixin-rewrite-tag-name' version '0.1.0'
[info]: gem 'fluent-mixin-type-converter' version '0.1.0'
[info]: gem 'fluent-plugin-parser-winevt_xml' version '0.2.5'
[info]: gem 'fluent-plugin-watch-process' version '0.2.0'
[info]: gem 'fluent-plugin-windows-eventlog' version '0.8.3'
[info]: gem 'fluent-plugin-windows-exporter' version '1.0.0'
[info]: Oj isn't installed, fallback to Yajl as json parser
[error]: config error in:
<source>
  @type windows_exporter
  tag "windows.metrics"
  scrape_interval 60
  cpu true
  memory true
  os trues
  net true
  logical_disk true
</source>

[error]: config error file="test.conf" error_class=Fluent::ConfigError error="'os' parameter is required but nil is specified"
```

**Comment by ashie** [comment]
Fixed in #19

**Comment by tikkss** [comment]
Thanks!
